tests/examples/test_format.py

```diff
@@ -31,7 +31,11 @@ def notebook_start_checker(dff_example_py_file: pathlib.Path):
     result = start_pattern.search("".join(file_lines))
     if result is None:
         raise Exception(
-            f"Example `{dff_example_py_file.relative_to(dff_examples_dir.parent)}` doesn't start as a notebook."
+            (
+                f"Example `{dff_example_py_file.relative_to(dff_examples_dir.parent)}` "
+                + "does not have an initial markdown section. Notebook header should be prefixed "
+                + "with a single '#'."
```

Comment:
The RegEx checks a lot of things actually, not only this one...

Reply:
If so, it would be easier if you documented them or added to this error message

Reply:
```suggestion
                + "with a single '# %% [markdown]'."
```